easybuild/easyconfigs/g/git-lfs/git-lfs-3.5.1.eb

```diff
@@ -0,0 +1,26 @@
+easyblock = 'GoPackage'
+
+name = 'git-lfs'
+version = '3.5.1'
+
+homepage = 'https://git-lfs.github.com'
+description = """Git Large File Storage (LFS) replaces large files such as audio
+ samples, videos, datasets, and graphics with text pointers inside Git, while 
+ storing the file contents on a remote server like GitHub.com"""
+
+toolchain = SYSTEM
+
+source_urls = ['https://github.com/git-lfs/git-lfs/archive/']
```

Comment:
Any reason for not keeping the `GITHUB_SOURCES` URL combined with `github_account=name` that's in prior EasyConfigs for `git-lfs`?

Reply:
good point, changed.